seed/api/v3/urls.py

```diff
@@ -69,6 +70,7 @@
 api_v3_router.register(r'postoffice', PostOfficeViewSet, basename='postoffice')
 api_v3_router.register(r'postoffice_email', PostOfficeEmailViewSet, basename='postoffice_email')
 api_v3_router.register(r'progress', ProgressViewSet, basename="progress")
+api_v3_router.register(r'data_loggers', DataLoggerViewSet, basename="data_logger")
```

Comment:
silly request, can you move this to be alphabetical? Looks like `data_quality_checks` is also out of order.